packages/modules/manage-versions/src/components/ManageVersions/ChangesTab/ChangesTab.scss

```diff
@@ -7,4 +7,19 @@
   .iui-table-body .iui-table-cell {
     word-break: break-word;
   }
-}
\ No newline at end of file
+
+  .iac-create-version-icon-hidden {
+    visibility: hidden;
+  }
+
+  .iac-changes-tab-actions-btn-group {
+    div {
+      width: 25px;
+    }
+    &:hover {
+      Button {
+        background-color: inherit;
```

Comment:
why this was needed? And I dont think `Button` is a valid selector...

Reply:
![image](https://github.com/iTwin/admin-components-react/assets/148448690/008671ec-a7c4-47ab-b6f6-b1be6ad37277)

 action buttons have dark background color when hover.
Since selectors in scss are not case-sensitive so it works with Button but it should be "button"

Reply:
please dont change the background color..
Can you just add the gap ebtween buttons of small size, and thats it?
Dont even use button group, cause its not for this purpose..

Reply:
removed ButtonGroup